database.ts

```diff
@@ -0,0 +1,329 @@
+/** @file Handles loading and storing data. */
+import sqlite from 'better-sqlite3'
+
+import * as newtype from './newtype.js'
+
+// =============
+// === Types ===
+// =============
+
+/** All possible emojis that can be used as a reaction on a chat message. */
+export type ReactionSymbol = '❤️' | '🎉' | '👀' | '👍' | '👎' | '😀' | '🙁'
```

Comment:
This is duplicated in another file. Please double check the code for duplications.

Reply:
this is intentional - not sure it makes too much sense to import `database.ts` from `chat.ts` or vice versa...
(f course there's no real problem, especially since it gets bundled anyway, and there should be no risk of a circular dependency since they don't *actually* depend on each other)

i could extract this (`ReactionSymbol`) to its own file, maybe?

Reply:
Yeah, if these two modules does not make sense to import each other, then refactoring it to another file is probably the best solution! :)